patterns/dswp-vertical-cards-with-icon.php

```diff
@@ -7,108 +7,233 @@
-                        <div class="wp-block-group alignfull" style="padding-top:0;padding-bottom:0"></div>
-                        <!-- /wp:group -->
+<!-- wp:group {"metadata":{"name":"DSWP Vertical Cards With Icon","categories":["columns"],"patternName":"design-system-wordpress-theme/dswp-vertical-cards-with-icon"},"className":"alignfull","style":{"spacing":{"margin":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="margin-top:var(--wp--preset--spacing--60);margin-bottom:var(--wp--preset--spacing--60)"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"default"}} -->
+    <div class="wp-block-group"><!-- wp:columns {"align":"wide","className":"dswp-vertical-card-with-icon","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|40","left":"var:preset|spacing|40"},"padding":{"right":"0","left":"0"}}}} -->
+        <div class="wp-block-columns alignwide dswp-vertical-card-with-icon" style="padding-right:0;padding-left:0"><!-- wp:column {"width":"","style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"0rem","right":"0rem"}},"border":{"radius":"7px","width":"1px","color":"var(\u002d\u002ddswp-surface-color-border-default)"}},"layout":{"type":"default"}} -->
+            <div class="wp-block-column has-border-color" style="border-color:var(--dswp-surface-color-border-default);border-width:1px;border-radius:7px;padding-top:0;padding-right:0rem;padding-bottom:0;padding-left:0rem"><!-- wp:columns {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|80","right":"var:preset|spacing|80"},"margin":{"top":"0","bottom":"0"}},"border":{"radius":"5px","top":{"color":"var(\u002d\u002ddswp-theme-primary-gold)","width":"9px"},"right":{},"bottom":{},"left":{}}},"backgroundColor":"background-light-gray"} -->
+                <div class="wp-block-columns has-background-light-gray-background-color has-background" style="border-radius:5px;border-top-color:var(--dswp-theme-primary-gold);border-top-width:9px;margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--80)"><!-- wp:column {"width":"100%"} -->
+                    <div class="wp-block-column" style="flex-basis:100%"><!-- wp:image {"id":16878,"sizeSlug":"full","linkDestination":"none","align":"full","className":"size-full is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"}}}} -->
+                        <figure class="wp-block-image alignfull size-full is-style-default" style="margin-top:0;margin-bottom:0"><img src="<?php echo esc_url( get_template_directory_uri() . '/assets/images/square-256.png' ); ?>" alt="" class="wp-image-16878" /></figure>
```

Comment:
it looks like this url is ok, however the colour markup of the php code is black and not (blue/red/purple), but can't seem to find a missing quote.  for example 
![image](https://github.com/user-attachments/assets/fffef48a-b2aa-4b79-8d27-c8d6c0aae367)